resources/authgear/templates/en/web/prompt_create_passkey.html

```diff
@@ -7,11 +7,13 @@ <h1 class="m-0 primary-txt text-center text-xl font-bold">
 	{{ template "prompt-create-passkey-page-title" }}
 </h1>
 
+<i class="authgear-icons authgear-icons-passkey text-3xl block primary-txt text-center" aria-hidden="true"></i>
+
 <div class="text-sm break-words primary-txt text-center">{{ template "prompt-create-passkey-page-description" }}</div>
 
 <div class="twc-container-vertical" data-controller="passkey-creation">
 
-<div class="flex flex-col space-y-2 my-5 items-center">
```

Comment:
Per our discussion in the production meeting, do we need to change the buttons to full width?

Reply:
Yes. Sorry.